library/src/compute/construct.rs

```diff
@@ -490,6 +499,71 @@ cast_from_value! {
     v: Str => Self(v),
 }
 
+/// Converts a character into its corresponding code point.
+///
+/// ## Example
+/// ```example
+/// #str.to-unicode("a") \
+/// #"a\u{0300}".codepoints().map(str.to-unicode)
```

Comment:
I guess the naming of `.codepoints()` is a bit confusing now, since it returns strings and not ints. Maybe we could change it, idk.